src/components/Landing/Landing.tsx

```diff
@@ -0,0 +1,122 @@
+import React from 'react'
+import classNames from 'classnames/bind'
+
+import style from 'components/Landing/Landing.module.scss'
+import ImgLogo from 'assets/img_logo.png'
+import ImgMain from 'assets/images/landingImg/img_main.png'
+import ImgLine from 'assets/images/landingImg/img_line.png'
+import ImgInviteIcon from 'assets/images/landingImg/img_inviteicon.png'
+import ImgCharacterChoice from 'assets/images/landingImg/img_characterchoice.png'
+import ImgMakeInvite from 'assets/images/landingImg/img_makeinvite.png'
+import ImgOnlineInvite from 'assets/images/landingImg/img_onlineinvite.png'
+import ImgChat from 'assets/images/landingImg/img_chat.png'
+import ImgReminder from 'assets/images/landingImg/img_reminder.png'
+
+const cx = classNames.bind(style)
+
+function Landing(props) {
```

Comment:
```suggestion
function Landing() {
```
props가 사용되는곳이 없어서 삭제해도 괜찮을것 같습니다